src/stream/src/executor/over_window/over_partition.rs

```diff
@@ -229,7 +230,9 @@ pub(super) struct OverPartition<'a, S: StateStore> {
     range_cache: &'a mut PartitionCache,
     cache_policy: CachePolicy,
 
-    calls: &'a [WindowFuncCall],
+    super_rows_frame_bounds: RowsFrameBounds,
+    start_is_unbounded: bool,
+    end_is_unbounded: bool,
```

Comment:
Do we need these? Seems like we can access them from `super_rows_frame_bounds`

Reply:
Later when we add `RANGE` frame, this will be required. `start_is_unbounded` is derived from **all** frames while `super_rows_frame_bounds` only from `ROWS` frames.